hieradata/class/share_sale_publisher.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+govuk::node::s_base::apps:
+  - share_sale_publisher
```

Comment:
We don't need new machines for these apps - they can go on the backend machines with other publishing tools.